### ISSUE TITLE
The specified web page cannot be closed.

### ISSUE THREAD

**Issue by jiqi136** [chromedp/chromedp#281]
The specified web page cannot be closed.

// CloseByIndex closes the Chrome target with specified index i.
func (c *CDP) CloseByIndex(i int) Action {
	return ActionFunc(func(ctxt context.Context, h cdp.Executor) error {
		return nil .????????
	})
}

// CloseByID closes the Chrome target with the specified id.
func (c *CDP) CloseByID(id string) Action {
	return ActionFunc(func(ctxt context.Context, h cdp.Executor) error {
		return nil .????????
	})
}

opts ...????????
func KeyAction(keys string, opts ...KeyOption) Action {
	return ActionFunc(func(ctxt context.Context, h cdp.Executor) error {
		var err error

		for _, r := range keys {
			for _, k := range kb.Encode(r) {
				err = k.Do(ctxt, h)
				if err != nil {
					return err
				}
			}

			// TODO: move to context
			time.Sleep(5 * time.Millisecond)
		}

		return nil
	})
}

**Comment by mvdan** [comment]
Sorry, this issue doesn't make any sense to me. Perhaps try explaining what you did in plain English, ideally with a full Go example.
